src/plugins/kibana_react/public/saved_objects/saved_object_finder.test.tsx

```diff
@@ -248,7 +248,7 @@ describe('SavedObjectsFinder', () => {
       });
     });
 
-    it('should include additional fields in search if listed in meta data', async () => {
+    it.skip('should include additional fields in search if listed in meta data', async () => {
```

Comment:
@flash1293 I'm not sure if you're the right person to ping about this (blame `git blame` 😅), but I had to skip these tests as they were producing unhandled promise rejections due to an uncaught exception.

Reply:
Here's the stack trace of the unhandled promise rejection:

```
TypeError: Cannot read property 'savedObjects' of undefined
    at SavedObjectFinder._lodash.default.debounce (./src/plugins/kibana_react/public/saved_objects/saved_object_finder.tsx:46:32)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Reply:
Should be fixed now, I pushed on this PR.

Reply:
Looking good! Thanks for helping out and providing the fix